docs/index-backend/field-mapping.md

```diff
@@ -120,4 +117,4 @@ mgmt.buildIndex('custom_property_test', Vertex.class).addKey(myProperty, Mapping
 mgmt.commit()
 ```
 
-With these settings, JanusGraph will use the boost 5 and boolean similarity algorithm for property key `my_property`. Possible mapping parameters depend on Elasticsearch version. See [mapping parameters](https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-params.html) for current Elasticsearch version.
\ No newline at end of file
+With these settings, JanusGraph will use the boost 5 and boolean similarity algorithm for property key `my_property`. Possible mapping parameters depend on Elasticsearch version. See [mapping parameters](https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-params.html) for current Elasticsearch version.
```

Comment:
(nitpick) GitHub shows a diff in this line, but I don't see anything that changed here (?)

Reply:
It is IDEA settings. Sometimes it adds `end of the line` hidden character. I will configure my IDEA to not change those characters automatically